SoapyFCDPP/SoapyFCDPP.cpp

```diff
@@ -513,31 +611,44 @@ SoapySDR::KwargsList findFCDPP(const SoapySDR::Kwargs &args)
     cur_dev = devs;
     while (cur_dev) {
         SoapySDR::Kwargs soapyInfo;
-        SoapySDR_logf(SOAPY_SDR_INFO, "Found device: %s", cur_dev->path);
         // This is the name that shows up.
-        soapyInfo["device"] = "Funcube Dongle Pro+";
+        soapyInfo["device"] = "Funcube Dongle Pro+ (192k)";
+        soapyInfo["hid_path"] = cur_dev->path;
+        soapyInfo["is_plus"] = "true";
+        soapyInfo["alsa_device"] = findAlsaDevice(cur_dev->path);
+        SoapySDR_logf(SOAPY_SDR_TRACE, "Found device: %s, %s", cur_dev->path, soapyInfo["alsa_device"].c_str());
+        cur_dev = cur_dev->next;
+        results.push_back(soapyInfo);
+    }
+    hid_free_enumeration(devs);
+    // now check for older Pro devices
+    devs = hid_enumerate(FCDPP_VENDOR_ID, FCD_PRODUCT_ID);
+    cur_dev = devs;
+    while (cur_dev) {
+        SoapySDR::Kwargs soapyInfo;
+        // This is the name that shows up.
+        soapyInfo["device"] = "Funcube Dongle Pro (96k)";
         soapyInfo["hid_path"] = cur_dev->path;
-        // TODO:
-        // * Currently only one dongle will work and I don't know how to associate
-        // * HID path with an alsa path. Perhaps this could be an option to the driver.
-        // * Perhaps make sample rate configureable.
-        soapyInfo["alsa_device"] = "hw:CARD=V20,DEV=0";
+        soapyInfo["is_plus"] = "false";
+        soapyInfo["alsa_device"] = findAlsaDevice(cur_dev->path);
+        SoapySDR_logf(SOAPY_SDR_TRACE, "Found device: %s, %s", cur_dev->path, soapyInfo["alsa_device"].c_str());
         cur_dev = cur_dev->next;
         results.push_back(soapyInfo);
     }
     hid_free_enumeration(devs);
     
+    SoapySDR_logf(SOAPY_SDR_INFO, "findFCDPP=%d devices", results.size());
```

Comment:
Similar comment here with the log level -- if its just going to print 0 devices every time the discovery is performed on a system

Reply:
As above, will review these.